### PR TITLE
Allow importer to use larger images

### LINKED ISSUE #608: Importer should use higher resolution posters
Somewhat similar to what I mentioned [here](https://forums.hummingbird.me/t/character-images-all-blurry/25712/2?u=synthtech), except for anime and manga posters.

Something that appends "l" to the filename will allow the importer to grab a (usually) better quality version of a poster.

Ex.

Thumbnail | Full-size
--- | ---
http://cdn.myanimelist.net/images/manga/2/161562.jpg | http://cdn.myanimelist.net/images/manga/2/161562l.jpg
 |

### PR DESCRIPTION
**Pull request by synthtech**
This should fix #608 and the [related issue](https://forums.hummingbird.me/t/character-images-all-blurry/25712)... I hope

### CODE REVIEW

File: lib/mal_import.rb [hunk 1]
```diff
@@ -154,6 +154,14 @@ def to_h
   end
 
   private
+  def character_image(img)
+    img = img.gsub("t.jpg", ".jpg")
+    if img.include?("questionmark")
+      nil
+    else
+      URI(img)
+    end
```
Comment (DNA) [inline lib/mal_import.rb]: you can change this `if` to:
```ruby
URI(img) unless img.include?("questionmark")
```
it automatically add `nil` if the last line is not executed :)

File: lib/mal_import.rb [hunk 2]
```diff
@@ -162,6 +170,14 @@ def person_image(img)
       URI(img)
     end
   end
+  def poster_image(img)
+    img = img.gsub(".jpg", "l.jpg")
+    if img.include?("na_series")
+      nil
+    else
+      URI(img)
+    end
```
Comment (DNA) [inline lib/mal_import.rb]: The same as the commentary above :)

**Comment by synthtech** [comment]
Finally fixed it. Thanks for the tip @DNA :smile:

**Comment by vevix** [comment]
Is it guaranteed that there will be an image with that suffix? Looks great otherwise

**Comment by synthtech** [comment]
Yeah, I've been checking a lot of entries and all of them have the larger source images

**Comment by vevix** [comment]
Awesome, I'll get this merged later tonight.

**Comment by vevix** [comment]
Um... Oops so this slipped my mind.

**Pull request merged**